cmake/boards_db.cmake

```diff
@@ -7130,6 +7130,114 @@ target_compile_options(GENERIC_C031F6PX_serial_none INTERFACE
   "SHELL:-DHAL_UART_MODULE_ENABLED -DHWSERIAL_NONE"
 )
 
+# GENERIC_C071R8TX
```

Comment:
Did you miss GENERIC_C071RBTX here as well?

Reply:
No if you look below you will see it.